utils/fieldToConceptIdMapping.js

```diff
@@ -390,10 +393,12 @@ module.exports = {
 
     dhq3SurveyStartTime: 109610692,
     dhq3SurveyCompletionTime: 610227793,
+    dhq3ResponseProcessedTime: 100000002, // TODO: update this
```

Comment:
Should we use the actual string (dhq3ResponseProcessedTime) instead of a fake CID, before geting the CID? I recall there was an incident that a fake CID looked so real that it was treated as an actual CID.